test/e2eclusterctl/create_workload_cluster_test.go

```diff
@@ -0,0 +1,37 @@
+// +build e2eclusterctl
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2eclusterctl
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Workoad cluster creation", func() {
+	CreateWorkloadClusterSpec(context.TODO(), func() CreateWorkloadClusterSpecInput {
```

Comment:
what is this test testing?

Reply:
This test is testing the creation and deletion of workload clusters through clusterctl, similar to what we are currently testing in test/e2e.

Reply:
FYI, in test/e2e we are encapsulating test specs in functions in order to allow re-use of the test spec.
If you don't have such a requirement for your specs (re-usability), you can avoid this function and implement your test directly in the Describe body